common/country_formation/00_major_formables.txt

```diff
@@ -62,11 +62,6 @@ ITA = {
 
 SCA = {
 	use_culture_states = yes
-	
-	is_major_formation = yes
-	
-	unification_play = dp_unify_scandinavia
-	leadership_play = dp_leadership_scandinavia
```

Comment:
why were we removing that?

Reply:
Because the Scandinavian formation kept dragging in other countries, I think. 
Fallonar wasn't happy with it being a major formation, at least.